.dockerignore

```diff
@@ -15,3 +15,5 @@ postgres-data/
 README.md
 node_modules/
 target/
+
+!docker/migrate/prisma/migrations/*
```

Comment:
Why do these have to be explicitly anti-ignored?

Reply:
Because they're included into the `attune_macros` crate at build time, so they have to be there when the `attune_macros` crate is built.

They were previously ignored on line 10 of the `dockerignore`.